hal/shared/demux.h

```diff
@@ -25,15 +25,19 @@
 #include "static_recursive_mutex.h"
 #include "pinmap_defines.h"
 #include "pinmap_hal.h"
+#if HAL_PLATFORM_NRF5280
 #include "nrf_gpio.h"
+#endif
 
 
 #define DEMUX_MAX_PIN_COUNT     8
+#if HAL_PLATFORM_NRF5280
```

Comment:
These pin assignments are particular to the Tracker One, not just nRF, so this should be `#if HAL_PLATFORM_TRACKER`

Reply:
Tracker One is a subset of nRF, right? so I have added `HAL_PLATFORM_NRF52840`. Is this pin config valid only for tracker one and not other nRF based platforms?

Reply:
For now there is no practical difference between using `HAL_PLATFORM_NRF52840` and checking for `PLATFORM_ID == PLATFORM_TRACKER`. Also note that there is no `HAL_PLATFORM_NRF5280` define.

Reply:
I will leave it as-is then. And fixed the typo there. 👍 🙂